main.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Karen Groener</title>
+  <link href="https://fonts.googleapis.com/css?family=Abril+Fatface|Italiana" rel="stylesheet" type='tex/css'>
+  <link rel="stylesheet" href="css/style.css" type="text/css">
+</head>
+
+<body>
+<header>
+</header>
+<body background="file:///Users/claudia/Desktop/FEWD/github/homework-01-famous-person-resume-website/imgs/fur_closeup.jpg" id="bg" alt="">
```

Comment:
Inline styling should be avoided due to issues with specificity (https://css-tricks.com/specifics-on-css-specificity/) and scale-ability. Also, this links to an image resource that does not exist. Please correct. 🔨

Also use relative links as opposed to absolute links, especially in this case where it is an absolute link to the file system as opposed to being a true URL.